cmd/antrea-agent/agent.go

```diff
@@ -103,7 +106,9 @@ func run(o *Options) error {
 		ifaceStore,
 		nodeConfig,
 		ovsconfig.TunnelType(o.config.TunnelType),
-		agentInitializer.GetIPSecPSK())
+		agentInitializer.GetIPSecPSK(),
+		routeClient,
+		agentInitializer.GeIPTablesClient())
```

Comment:
Humm. would you consider to move the IPTablesClient creation to this file just like route.NewClient() in line 72?